spec/models/user_spec.rb

```diff
@@ -297,4 +297,25 @@
       expect { do_purge }.to change(User, :count).by(0)
     end
   end
+
+  #sam Unit test
+  describe User do
+    let(:emails) {  ['a@a.com','b@b.com'] }
+    let(:users) { double('Users') }
+        it '.superadmin_emails' do 
+          expect(users). to receive(:pluck).with(:email).and_return(emails)
+          expect(User).to receive(:superadmins).and_return(users)
+          expect(User.superadmin_emails).to eq emails
+        end
+  end
+
+  #integration test 
```

Comment:
This will be a great resource: https://pragprog.com/book/rspec3/effective-testing-with-rspec-3 I'm asking the author if we can get discounted copies for AV premiums, or premium mobs ...